privacy.html

```diff
@@ -0,0 +1,91 @@
+      depends on the Personal Information we collect and the specific context in which we collect the information:</p>
+    <ul>
+      <li>Two Factor Auth needs to perform a contract with you</li>
+      <li>You have given Two Factor Auth permission to do so</li>
+      <li>Processing your personal information is in Two Factor Auth legitimate interests</li>
+      <li>Two Factor Auth needs to comply with the law</li>
+    </ul>
+    <p>Two Factor Auth will retain your personal information only for as long as is necessary for the purposes set out
+      in
+      this Privacy Policy. We will retain and use your information to the extent necessary to comply with our legal
+      obligations, resolve disputes, and enforce our policies.</p>
+    <p>If you are a resident of the European Economic Area (EEA), you have certain data protection rights. If you wish
+      to
+      be informed what Personal Information we hold about you and if you want it to be removed from our systems, please
+      contact us.</p>
+    <p>In certain circumstances, you have the following data protection rights:</p>
+    <ul>
+      <li>The right to access, update or to delete the information we have on you.</li>
+      <li>The right of rectification.</li>
+      <li>The right to object.</li>
+      <li>The right of restriction.</li>
+      <li>The right to data portability</li>
+      <li>The right to withdraw consent</li>
+    </ul>
+    <h2>Data Access</h2>
+    <p>The only parties that have access to the data include Josh Davis (Seattle, WA) as well
+      as Google LLC (1600 Amphitheater Parkway, Mountain View, CA 94043, USA)</p>
+    <h2>Google Analytics</h2>
+    <p>The services contained in this section enable the Owner to monitor and analyze web traffic and can be used to keep track of User behavior. </p>
+    <p>Google Analytics is a service that analyzes web traffic and is provided by Google LLC (“Google”). The data is collected and used to prepare
+      reports on user activity and shares it with Google's services. The data collected includes but not limited to: referrer, page visit data
+      such as time of visit, duration of visit, web browser and OS details. Data is anonymized.</p>
+    <p>View more information about Google Analytics here: <a href="https://analytics.google.com/analytics/web/">Google Analystics Info</a></p>
+    <p>More details:</p>
+    <ul>
+      <li><strong>Data process location:</strong> United States</li>
+      <li><strong>Personal Data processed:</strong> Tracker, usage data</li>
+    </ul>
+    <h2>Cookies and Web Beacons</h2>
+    <p>Like any other website, Two Factor Auth uses 'cookies'. These cookies are used to store information including
+      visitors' preferences, and the pages on the website that the visitor accessed or visited. The information is used
+      to
+      optimize the users' experience by customizing our web page content based on visitors' browser type and/or other
+      information.</p>
```

Comment:
Website is static
```suggestion
```

Reply:
Sure, but I think that's irrelevant from a technical/legal standpoint.